compiler-core/src/language_server/tests/completion.rs

```diff
@@ -19,13 +23,33 @@ fn positioned_expression_completions(
     src: &str,
     dep: &str,
     position: Position,
+) -> Vec<CompletionItem> {
+    positioned_expression_completions_with_dep(src, dep, false, position)
+}
+
+/// `external` is used to tell whether the specified dependency has to be
+/// treated as coming from an external imported package or as coming from a
+/// module from the same root package.
+/// In any case `src` will be able to import it using `import dep`.
```

Comment:
External means it's written in FFI, dependency means it comes from another package. I think with the old name it should have always been in a dep package.

Reply:
The way it was implemented it was just considered as another module under the same root package. I can change it to make sure the dep is always under another package if you want